platform/src/apis/Platform.Api.Establishment/Features/Schools/Services/SchoolsService.cs

```diff
@@ -67,4 +68,24 @@ public Task<SuggestResponse<School>> SuggestAsync(SchoolSuggestRequest request)
                 : $"({string.Join(") and ( ", request.Exclude.Select(a => $"URN ne '{a}'"))})";
         }
     }
+
+    public Task<SearchResponse<School>> SchoolsSearchAsync(SearchRequest request)
+    {
+        var facets = new[]
+        {
+            nameof(School.OverallPhase),
+        };
+
+        var response = SearchAsync(request, CreateSearchFilterExpression, facets);
+
+        return response;
+
+        string? CreateSearchFilterExpression(FilterCriteria[]? filterCriteriaArray)
```

Comment:
(Note to self: for consistency, move to `SearchRequest.FilterExpression()` as per `SchoolSuggestRequest`)